src/player.js

```diff
@@ -1508,6 +1504,16 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  _handleFrameDrop(event: FakeEvent): void {
+    const videoTracks = ((this._getTracksByType(TrackType.VIDEO): any): Array<VideoTrack>);
+    const sortedVideoTracks = videoTracks.sort((trackA, trackB) => trackA.bandwidth - trackB.bandwidth);
+    const currentBandwidthTrackIndex = sortedVideoTracks.findIndex(track => track.bandwidth && track.bandwidth === event.payload.bandwidth);
+    // not out of the array boundaries
+    if (currentBandwidthTrackIndex - 1 >= 0) {
+      this.selectTrack(sortedVideoTracks[currentBandwidthTrackIndex - 1]);
```

Comment:
This doesn't enable adaptive switching, but just select a specific video track, which wasn't the intended use case.
The requirement is for adaptivness to still be retained while marking a level as not allowed for use due to high FPS.